views/components/SignUpForm.js

```diff
@@ -1,39 +1,59 @@
 import React from 'react';
+import AlphaSignUpForm from './AlphaSignUpForm';
+import BetaSignUpForm from './BetaSignUpForm';
 import FormField from './FormField';
 
 const SignUpForm = props => {
-  const { header, subhead, partnerId } = props;
+  const { header, subhead, partnerId, hideAlpha, showBeta } = props;
```

Comment:
Yea maybe. I went this route because by default the alpha form should be shown and the beta form should be hidden. And it then leaves the onus on the campaign/config creator to specify if some non-default behavior's needed. But maybe the values should just be required for all partnerships?